### PR TITLE
Rs 2022 04 03

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 1]
```diff
@@ -203,12 +215,13 @@ It would be dull to just get water. I wonder if that ice cream van is still ther
 [sha:0d8bbf97c881ed0afb6d572328bb7b7902befd38]{
 -- Page 260, Offset 24915.
 -- 正午過ぎになっても、俺たちは公園から移動する事はなかった。
-Even as it turned past noon, we didn't move from our spot in the park.
+Even after the morning gave way to the afternoon, we didn't move from our spot in the park.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:218]: after morning gave way to afternoon

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt [hunk 2]
```diff
@@ -463,32 +484,37 @@ There would be something missing in the process before acting, right? Or rather,
...
 -- アルクェイドの視線の先には遊んでいる子供たちがいるから、彼女の聞きたい事はそういう事なんだろう。
-There are children playing around in her line of sight, so that must be why she's asking me that.
+She's watching some children playing around in front of her,
+ so presumably that's what she's asking about.
 }
 [sha:62859116c54074465f32a293310815bdebcc3413]{
 -- Page 282, Offset 24972.
 -- 「さあ、どうかな。たしかに子供のころは遊びまわってたけど、それは屋敷の中だけの話でさ。こういう公園で遊んだ事は少なかった」
-"Who knows? It's true that I played around when I was a kid, but it was only inside the mansion. I never really played around in a park like this much."
+"Hmm, hard to say.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_3_3.txt:509]: "hard to say" is a little less flippant than "who knows" so i say revert

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_3_3.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_3_3.txt:509]: Hmm, I thought this was more musing than flippant. I'll go back and re-listen.

Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_3_3.txt:509]: Hm yeah it sounds more considered to me

Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_3_3.txt:509]: alright

**Pull request merged**